mathics/builtin/forms/output.py

```diff
@@ -850,12 +854,9 @@ class StandardForm(FormBaseClass):
     </dl>
 
     >> StandardForm[a + b * c]
-     = a + b c
+     = a+b c
```

Comment:
With our current implementation, `StandardForm` does not add spaces around the operators. If we change `MakeBoxes[Infix[...],...]`   to allow the spaces, then TeXForm and MathMLForm get broken.
Spaces should be added at the level of `boxes_to_format` routines, probably controlled by a certain parameter in the `RowBox` object. But this is something for another round.